### PR TITLE
remove Gitosis

### PR DESCRIPTION
**Pull request by pismute**
Progit2 no longer talks about Gitosis so that it is little bit wierd.

### CODE REVIEW

**Comment by ben** [comment]
Nice catch. Thanks!

File: book/04-git-server/sections/git-daemon.asc [hunk 1]
```diff
@@ -41,7 +41,7 @@ respawn
 ----
 
 For security reasons, it is strongly encouraged to have this daemon run as a user with read-only permissions to the repositories – you can easily do this by creating a new user 'git-ro' and running the daemon as them.
-For the sake of simplicity we'll simply run it as the same 'git' user that Gitosis is running as.
+For the sake of simplicity we'll simply run it as the same 'git' user that `git-shell` is using as.
```
Comment (michaelgwelch) [inline book/04-git-server/sections/git-daemon.asc]: I know this is closed. I just stumbled upon this while brwosing. Is 'using as' the right phrase? 'running as' seemed better. It doesn't seem grammatical now.

Comment (ben) [inline book/04-git-server/sections/git-daemon.asc]: Woops, that slipped past me. Wanna correct it?

**Comment by michaelgwelch** [comment]
Sure see #234 

Probably I should have checked contribution guidelines. Let me know if commit message needs to be improved.

**Pull request merged**